app/assets/javascripts/application.js

```diff
@@ -51,8 +51,8 @@
 //= require controllers/container_dashboard/card/card-module
 //= require controllers/container_dashboard/card/heatmaps/heatmaps-card-directive
 //= require controllers/container_topology/container_topology_controller
-//= require d3/d3
-//= require c3/c3
+//= require c3
+//= require d3
```

Comment:
https://github.com/rails/sprockets part "Index files are proxies for folders". This way seems to be consistent with the docs.